src/main/java/io/inisos/bank4j/impl/JAXBCreditTransferV09.java

```diff
@@ -253,9 +261,11 @@ private Optional<BranchAndFinancialInstitutionIdentification6> optionalBranchAnd
     private BranchAndFinancialInstitutionIdentification6 mandatoryBranchAndFinancialInstitutionIdentification(BankAccount bankAccount) {
         BranchAndFinancialInstitutionIdentification6 branchAndFinancialInstitutionIdentification = new BranchAndFinancialInstitutionIdentification6();
         FinancialInstitutionIdentification18 financialInstitutionIdentification = new FinancialInstitutionIdentification18();
-        bankAccount.getBic().ifPresent(bic -> {
+        bankAccount.getBic().ifPresentOrElse(bic -> {
             BicUtil.validate(bic);
             financialInstitutionIdentification.setBICFI(bic);
+        }, () -> {
+            throw new IllegalArgumentException("BIC must be provided for debtor account");
```

Comment:
Looking at the XSD, there is no difference between V03 and V09:
Yes `FinInstnId` is mandatory, but the XSD allows it to be empty.
In other terms, it does not mandate `BICFI`, nor other child elements:

- `ClrSysMmbId`
- `LEI`
- `Nm`
- `PstlAdr`
- `Othr`

So I'm not comfortable with throwing `IllegalArgumentException` when an apparently optional field is missing.

Or did I miss something?